src/providers.ts

```diff
@@ -284,7 +284,7 @@ export function generateHoverString(flag: FeatureFlag, c: FlagConfiguration, url
 	hoverString.appendMarkdown(`${Prereqs} ${targets} ${rules}`);
 	hoverString.appendText('\n');
 	hoverString.appendMarkdown('**Variations**');
-	flag.variations.map((variation, idx) => {
+	c.variations.map((variation, idx) => {
```

Comment:
Just saw this, We actually want to loop over the Flag Variations, because that has the Name & Description available. Unless this is part of whole reorg we talked about.